CampusTour/Helpers/SearchHelpers.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  SearchHelpers.swift
+//  CampusTour
+//
+//  Created by Ji Hwan Seung on 4/9/18.
+//  Copyright © 2018 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class SearchHelper {
+    
+    //Takes in tag as input (it is the abbreviated form of the tags)
+    static func getEventsFromTag(tag: String) -> [Event] {
+        var tagid: String = ""
+        
+        generalTagMapping.forEach { tagid = $0.value == tag ? $0.key : tagid }
+        
+        var taggedEvents = [Event]()
+        
+        for e in DataManager.sharedInstance.events {
```

Comment:
I'm thinking we should pass in `DataManager.sharedInstance.events` as a parameter `events`, so that we can handle any data changes from the VC rather than the helper class

Reply:
I agree 😺